### PR TITLE
Support HerdDB database

### PR DESCRIPTION
**Pull request by eolivelli**
- Add HerdDB JDBC driver (works for tests, accessing a standalone server, accessing a cluster, running the server in the same process of the backend)
- Change all of the DELETE statements to use a syntax supported by Apache Calcite (SQL interpreter of HerdDB)
- Use time_stamp in SQL, and getTimestamp() in beans (there is no way to have a unique sintax to escape keyword 'timestamp' that works for MySQL/HerdDB and PostGre)
- Run tests with HerdDB instead of SQLLite
- add demo configuration for HerdDB
- upgrade Lombok

### CODE REVIEW

**Comment by eolivelli** [comment]
@tuteng I still have a failing test so this patch is not ready to go.

I also have a bad problem with the aliases 'timestamp', that is not supported by Apache Calcite.
When we were MySQL-friendly we could use the 'accent' escape of MySQL (´) but I think this is not supported by PostGres.
I think you used all the 'AS' syntaxes in order to map 1:1 the query to the java beans without renaming timestamp to time_stamp every where in the frontend.

I am trying to find a fix, byt if you have an idea about how to fix it please chime in

cc @sijie

**Comment by sijie** [comment]
@eolivelli what is the difference between #183 and this one?

**Comment by eolivelli** [comment]
@sijie I had to start from scratch because of too many conflicts as the schema changes dramatically and so every SQL query was rewritten.
The only big addition is that I have moved the tests to HerdDB.

**Comment by eolivelli** [comment]
@sijie  @tuteng all tests are passing.
The frontend seems to work even if I have renamed 'timestamp' to 'time_stamp' on every Stat bean.
I tried to check for direct usages of property "frontend" in the JS code but I did not find any explicit reference.

**Comment by eolivelli** [comment]
I had to upgrade Lombok in order to make my IDE happy, I can revert that part of the change

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -25,12 +25,23 @@ logging.level.io.streamnative=DEBUG
...
+# SQLLite
+#spring.datasource.driver-class-name=org.sqlite.JDBC
+#spring.datasource.url=jdbc:sqlite:pulsar_manager.db
+#spring.datasource.initialization-mode=always
+#spring.datasource.schema=classpath:/META-INF/sql/sqlite-schema.sql
+#spring.datasource.username=
+#spring.datasource.password=
+
+# HerdDB JDBC Driver
+ spring.datasource.driver-class-name=herddb.jdbc.Driver
```
Comment (eolivelli) [inline src/main/resources/application.properties]: this is now the default

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -25,12 +25,23 @@ logging.level.io.streamnative=DEBUG
...
+ HerdDB - local in memory-only (check docs for starting an embedded server or embedded replicated server)
+ spring.datasource.url=jdbc:herddb:local
```
Comment (eolivelli) [inline src/main/resources/application.properties]: this is in-memory only

**Comment by eolivelli** [comment]
@sijie  @tuteng the patch is good to go

**Comment by sijie** [comment]
The change looks good to me. @tuteng can you please also take a look?

**Comment by tuteng** [comment]
@eolivelli  Thank you very much for your contribution. There is no problem with the tests on my side. It works very well. Is it convenient for you to add some instructions on the default test database to this document https://github.com/streamnative/pulsar-manager/blob/master/src/README.md? 

Similar to the following:

### Default Test database HerdDB

#### Introduction

A brief introduction

Document Link: herddb-link

**Pull request merged**